### PR TITLE
[Feature] support host_name()

### PR DESCRIPTION
**Pull request by fzhedu**
Signed-off-by: fzhedu <fzhedu@gmail.com>

## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #
to get host name of the node

```
mysql>  select host_name(), max(siteid) as a from t  group by pv;
+-------------+------+
| host_name() | a    |
+-------------+------+
| sandbox-sql |   23 |
| sandbox-sql |   25 |
| sandbox-sql |   24 |
+-------------+------+
3 rows in set (0.09 sec)

mysql> select host_name();
+-------------+
| host_name() |
+-------------+
| sandbox-sql |
+-------------+
1 row in set (0.01 sec)
```

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [x] I have added user document for my new feature or new function https://github.com/StarRocks/docs.zh-cn/pull/1943

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by imay** [comment]
@fzhedu First, thanks for you contribution. I have a question, starrocks is a distributed system, often one cluster will contain many nodes and what does the host_name() stand for?

**Comment by fzhedu** [comment]
> @fzhedu First, thanks for you contribution. I have a question, starrocks is a distributed system, often one cluster will contain many nodes and what does the host_name() stand for?

when you want to know something related to each BE, this function works. for example, getting how many rows does each BE process, you can use
```
select h, count(id) from (select host_name() as h, id from t) group by h;
```

File: be/src/exprs/vectorized/utility_functions.cpp [hunk 1]
```diff
@@ -235,4 +236,15 @@ ColumnPtr UtilityFunctions::assert_true(FunctionContext* context, const Columns&
     return ColumnHelper::create_const_column<TYPE_BOOLEAN>(true, size);
 }
 
+ColumnPtr UtilityFunctions::host_name(starrocks_udf::FunctionContext* context, const Columns& columns) {
+    static std::string host_name;
```
Comment (mofeiatwork) [inline be/src/exprs/vectorized/utility_functions.cpp:240]: why static? what if concurrent access?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by kangkaisen — APPROVED** [review]

**Comment by kangkaisen** [comment]
please add document.

**Review by murphyatwork — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: be/src/exprs/vectorized/utility_functions.cpp (hunk 1 above)
Comment (fzhedu) [reply be/src/exprs/vectorized/utility_functions.cpp]: fixed

**Comment by fzhedu** [comment]
run starrocks_admit_test

**Comment by fzhedu** [comment]
run starrocks_admit_test

**Comment by fzhedu** [comment]
rerun starrocks_admit_test

**Comment by fzhedu** [comment]
run starrocks_admit_test

**Comment by fzhedu** [comment]
run starrocks_admit_test

**Comment by fzhedu** [comment]
run starrocks_admit_test

**Pull request merged**